### PR TITLE
Metric dependency

### PR DESCRIPTION
**Pull request by Olenek**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+intellij {
+    type = "IC"
+    version = "2021.1"
+    setPlugins("java")
+}
+
+group = "org.jetbrains.research.anticopypaster"
+version = "3.3"
```
Comment (onewhl) [inline build.gradle.kts]: I suggest using the following version naming convention:
'A-B'
A - version of IDE (for example 2021.3)
B - version of the plugin (2.0)

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    compile("org.apache.commons:commons-lang3:3.0")
+    implementation("org.pmml4s:pmml4s_2.13:0.9.10")
+
+    // extract-method-metrics module inclusion
+    implementation("$extractMethodProjectName:extract-method-metrics") {
+        version {
+            branch = "plugin-utils" // until merged into master
+        }
+    }
+
```
Comment (onewhl) [inline build.gradle.kts]: extra line

File: build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,42 @@
+plugins {
+    java
+    id("org.jetbrains.intellij") version "0.7.2"
+}
+
+intellij {
+    type = "IC"
+    version = "2021.1"
```
Comment (onewhl) [inline build.gradle.kts:8]: let's update to 2021.2 and check if everything works fine

**Review by onewhl — CHANGES_REQUESTED** [review]
Anton, thank you! Everything looks good, please, take a look at the couple of minor comments.

File: build.gradle.kts (hunk 1 above)
Comment (Olenek) [reply build.gradle.kts]: changed

File: build.gradle.kts (hunk 2 above)
Comment (Olenek) [reply build.gradle.kts]: removed

File: build.gradle.kts (hunk 3 above)
Comment (Olenek) [reply build.gradle.kts:8]: everything is not fine(

**Pull request merged**